cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/NoRequestOrdererV3.java

```diff
@@ -0,0 +1,20 @@
+package org.cloudfoundry.client.lib.util;
+
+public class NoRequestOrdererV3 implements RequestOrdererV3 {
+
+    @Override
+    public String orderBy() {
+        return null;
+    }
+
+    @Override
+    public OrderDirectionV3 direction() {
+        return null;
```

Comment:
I'm not sure, but I think we should return "ascending" here. If you specify no `order_by` query parameter, then the API uses default ordering and the default direction is "ascending".

Reply:
Nothing happens - tested it. :D